packtivity/handlers/execution_handlers.py

```diff
@@ -177,7 +177,15 @@ def docker_execution_cmdline(config, state, log, metadata, race_spec):
     cid_file = "--cidfile {}".format(cidfile)
 
 
-    env_vars = ' '.join([f"-e {env_entry['name']}='{env_entry['value']}'" for env_entry in race_spec['env']])
+    if "env" in race_spec:
```

Comment:
This just checks to see if `env` exists in the `race_spec` dict so that it doesn't give a KeyError is if doesn't.